build.gradle

```diff
@@ -154,6 +154,17 @@ subprojects {
         }
     }
 
+    task generateProperties(type: WriteProperties) {
```

Comment:
Could we also find a way to add any new properties to the manifest? https://github.com/google/conscrypt/blob/master/openjdk/build.gradle#L153

Reply:
oh just saw your comment.  It's fine as-is.